content/blog/2018/06/2018-05-14-simple-pull-request-plugin.adoc

```diff
@@ -125,42 +133,136 @@ using respective paltfroms for their repositories.
 is written which will load the "Jenkinsfile.yaml" and make use of PipelineSnippetGenerator class
 to generate Declarative pipeline code.
 . Reporting of results.
-. Plugin is using Yaml from target branch right now. (Maybe this needs some discussion, example: 
+. Plugin is using Yaml from target branch right now. (Maybe this needs some discussion, example:
 what if PR contains changes in Jenkinsfile.yaml)
-. Git Push step: To push the changes of PR to the target branch. This is implemented  
+. Git Push step: To push the changes of PR to the target branch. This is implemented
 using git-plugin, PushCommand is used for this from git-plugin. credentialId,
 branch name and repository url for intracting with Github, Bitbucket, etc
 will be taken automatically from respective plugins plugin. (Yes users have to configure in that way, you can see
-link:https://github.com/jenkinsci/simple-pull-request-job-plugin/blob/master/README.md[How to run the demo])
+link:https://github.com/Jenkinsci/simple-pull-request-job-plugin/blob/master/README.md[How to run the demo])
 
 === Jenkinsfile.yaml example
 
 Write now no format has been decided for the yaml file. But for this demo I am using below Jenkinsfile.yaml
```

Comment:
I would rephrase this: "For the phase 1 prototype demonstration, the following yaml file was used. Note that this format is subject to change in the next phases of the project, as we formalise the yaml format definition."